packages/providers/mgt-teamsfx-provider/src/TeamsFxProvider.ts

```diff
@@ -27,7 +27,7 @@ export class TeamsFxProvider extends IProvider {
   }
 
   /**
-   * returns teamsfx instance
+   * returns teamsfx instance, if you construct TeamsFxProvider with TeamsUserCredential, this value should be null
```

Comment:
Is there any reason that this `teamsfx` property needs to be public?
Perhaps it should move to a private property in v3 (I don't want to do this in the current release as it might break some existing customers)

Reply:
Yes, you're right, to avoid breaking current user's code, so just keep it public as before.

And for teamsfx SDK, and we recommend user to use TeamsUserCredential instead of teamsfx, and for furture release in v3, we plan to remove teamsfx class inside this auth provider.

According to @sebastienlevert , you're planning to release v3 towards the end of this year, and I will commit my changes before that for the branch `next/fluentui`